server/models/database.sql

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE users (
```

Comment:
Consider adding the `IF NOT EXISTS` specifiers

Reply:
CREATE TABLE will not overwrite an existing relation. IF NOT EXISTS only turns off the error message. I have decided that I want to see the error message.